### ISSUE TITLE
Problem in languages with some link with low wink "_" replaced with "<em>" and "</em>"

### ISSUE THREAD

**Issue by adriantintpilver** [breck7/pldb#340]
i don't know, if this is a problem recently or are there for a long time, attaching print screens for the steps

I see the .pldb its ok
![iota-lang-pldb](https://user-images.githubusercontent.com/114493221/203866408-d0990f40-cedc-434a-ad15-9aa2a1a9ae95.jpeg)

the .scroll its ok too
![iota-lang-scroll](https://user-images.githubusercontent.com/114493221/203866432-fb2e253e-43f1-4796-8c41-63de982d35e6.jpeg)

but in the .html the problem is there, in some place of build replaced the "_" for "<em>" and "</em>" tags.
![iota-lang-html](https://user-images.githubusercontent.com/114493221/203866438-21d48e50-d19f-433f-8a70-5b1138b5b489.jpeg)

and in the pldb.com in production the problem is in there's too.
![iota-lang-html-PLDB-PRD RED Alert](https://user-images.githubusercontent.com/114493221/203866947-1656fec9-0a4e-422b-8fe0-ca96ae3cf95c.jpeg)

and the link opens broken
![iota-lang-wiki-link-broken](https://user-images.githubusercontent.com/114493221/203867575-40a16302-ad6f-4f9d-8b34-1efb3d382548.jpeg)

I will continue to analyze this, but any ideas are welcome.

**Comment by breck7** [comment]
Wow! Great spot @adriantintpilver ! This is a bug with scroll, where scroll is matching "_" inside of quotes. I think I had something like this earlier too. There's a simple fix. I can't get to it right at the moment, so feel free to play around, otherwise I'll fix later. Nice catch!

**Comment by breck7** [comment]
I did 2 things to fix this:

* I removed my custom `nameToAnchor` method and just use `lodash.camelCase`. That makes a number of things better and is less code!
* I removed some HTML and just use Scroll more.

I hope that solves all of these instances. The alternative was adding HTML parsing code in Scroll to not replace `_` when inside quotes, but that is a lot more complex and I think we can avoid needing that.
